eZ/Publish/Core/Search/Common/FieldValueMapper/BooleanMapper.php

```diff
@@ -32,7 +32,7 @@ public function canMap(Field $field)
     }
 
     /**
-     * Map field value to a proper Elasticsearch representation.
+     * Map field value to a proper search engine representation.
```

Comment:
It depends on the engine, for example geo location field, common (from Solr, that is taken as a base) and ES:
- [eZ/Publish/Core/Search/Common/FieldValueMapper/GeoLocationMapper.php](https://github.com/ezsystems/ezpublish-kernel/blob/f593480c1216a8cf8506184d29a7fce22c97ae4f/eZ/Publish/Core/Search/Common/FieldValueMapper/GeoLocationMapper.php)
- [eZ/Publish/Core/Search/Elasticsearch/Content/FieldValueMapper/GeoLocationMapper.php](https://github.com/ezsystems/ezpublish-kernel/blob/f593480c1216a8cf8506184d29a7fce22c97ae4f/eZ/Publish/Core/Search/Elasticsearch/Content/FieldValueMapper/GeoLocationMapper.php)

I'll reword to reflect that.